ui/apps/platform/src/Containers/MainPage/Navigation/NavigationItem.tsx

```diff
@@ -11,7 +11,7 @@ export type NavigationItemProps = {
 function NavigationItem({ isActive, path, content }: NavigationItemProps): ReactElement {
     return (
         <NavItem isActive={isActive}>
-            <NavLink exact to={path} activeClassName="pf-m-current">
+            <NavLink exact to={path} className={isActive ? 'pf-m-current' : ''}>
```

Comment:
We cannot rely on the deprecated `activeClassName` prop here, as it will return true if the path matches _but the URL parameters are different_.

Note that this will cause a conflict with the react-router upgrade changes [here](https://github.com/stackrox/stackrox/pull/13628/files#diff-8fa75f66f768fcf9bd30ee302217e14630874315a9f202c40595a306a9f00e64) that we will need to evaluate.